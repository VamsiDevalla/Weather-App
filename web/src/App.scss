@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
@import './colors';

body {
  font-family: 'Roboto Slab', serif;
  background-color: $secondary-color;
}

* {
  box-sizing: border-box;
}

.App {
  color: $primary-color;
  height: 100vh;
  padding: 25px;
}

@media only screen and (max-width: 800px) {
  .App {
    padding: 10px;
  }
}
