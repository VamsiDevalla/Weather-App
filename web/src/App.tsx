import React from 'react';
import './App.scss';
import Homepage from './pages/homepage/homepage.component';

function App() {
  return (
    <div data-testid='app-container' className='App'>
      <Homepage />
    </div>
  );
}

export default App;
