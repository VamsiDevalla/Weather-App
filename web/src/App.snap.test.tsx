import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';


jest.mock('./pages/homepage/homepage.component',  () => "Homepage")
describe("App", () => { 
  afterAll(() => jest.clearAllMocks()); 
  test("should render correctly", () => {
    const app = renderer.create(<App/>).toJSON();
    expect(app).toMatchSnapshot();
  })
})
