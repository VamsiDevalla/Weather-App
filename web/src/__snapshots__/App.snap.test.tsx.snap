// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  className="App"
  data-testid="app-container"
>
  <Homepage />
</div>
`;
